tests/storage/imageupload.go

```diff
@@ -203,15 +203,29 @@ var _ = SIGDescribe("[Serial]ImageUpload", func() {
 		DescribeTable("Should succeed", func(resource, targetName string, validateFunc func(string), deleteFunc func(string)) {
 			defer deleteFunc(targetName)
 
-			By("Upload image")
-			virtctlCmd := tests.NewRepeatableVirtctlCommand("image-upload",
+			args := []string{
+				"image-upload",
 				resource, targetName,
 				"--namespace", util.NamespaceTestDefault,
 				"--image-path", imagePath,
 				"--size", pvcSize,
 				"--force-bind",
-				"--block-volume",
-				"--insecure")
+				"--insecure",
+			}
+
+			if sc, exists := tests.GetCephStorageClass(); exists {
+				args = append(args,
+					"--storage-class", sc,
+					"--block-volume",
+				)
+			} else {
+				args = append(args,
+					"--access-mode", "ReadWriteOnce",
```

Comment:
The only idea of this test is that when it runs on a storage with WFFC binding mode, and then CDI can force the bonding of the volume by starting worker nodes. On Immidiate storage this is basically a noop. 

Does ceph/block storage class use bindingMode = WFFC? If yes then ok (but maybe it needs a comment) if not then we could just skip the test.

Now I think, the test should "find" a storage class with a correct binding mode and only run on that storageClass.

Reply:
Right, now I am thinking about why there are no errors in the CI with ceph. Probably it does not use WFFC. I tihnk I can introduce some logic here to ensure an appropriate sc with WFFC is used.

Reply:
It also look that I did not check this correctly, when introducing the test

Reply:
Updated the test so it now uses local wffc sc. Also checked that ceph indeed has volume binding mode set to `Immediate`. Now this should work in all cases whether ceph is present or not.